### PR TITLE
oops. update shas too

### PR DESCRIPTION
**Pull request by nerdrew**
I removed the mavericks bottle shas, don't think anyone is running mavericks anymore though.

cc @xaviershay

### CODE REVIEW

**Comment by xaviershay** [comment]
dammit I was going to comment on that but didn't. PR fail.

File: gssh.rb [hunk 1]
```diff
@@ -18,6 +18,5 @@ def install
   bottle do
     root_url 'https://raw.githubusercontent.com/square/homebrew-bottles/master'
     cellar :any
-    sha256 "b329fe9b2f8248b2854b3eb0ebed16a52e24292c" => :mavericks
```
Comment (xaviershay) [inline gssh.rb]: this one doesn't have any SHAs anymore?

Comment (nerdrew) [inline gssh.rb]: I added mavericks to erg, but I don't even see a gssh-1.0.1 bottle in https://github.com/square/homebrew-bottles for _any_ OS version

Comment (nerdrew) [inline gssh.rb]: Am I missing something?

Comment (xaviershay) [inline gssh.rb]: yeah weird, maybe someone released a new version and didn't publish a bottle? Eh, merging.

**Pull request merged**